m2cgen/exporters.py

```diff
@@ -104,6 +104,70 @@ def export_to_javascript(model, indent=4):
     return _export(model, interpreter)
 
 
+def export_to_visual_basic(model, module_name="Model", indent=4):
+    """
+    Generates a Visual Basic (also can be treated as VBA
+    with some small manual changes, see a note below)
+    code representation of the given model.
+
+    .. note::
```

Comment:
I wonder whether this note should belong to the FAQ section of the readme

Reply:
Or maybe it should be in both places: here and there?.. I think some users do not tend to read any sections of READMEs except API reference, they finish reading on docstrings.

Reply:
It just seems unlikely that anyone will look up a guide on how to generate code for VBA in API docs to VB generator.

Reply:
Maybe! But I thought this note here can be useful in following scenarios:
- help to discover that it possible to generate VBA for those who use VB;
- some users might come here after failed search for `export_to_vba` as VBA is a dialect of VB;
- in case someone in the future modify/update this code, the note helps to indicate that code should be compatible with both VB and VBA.